src/components/member-profile/index.js

```diff
@@ -372,6 +372,28 @@ const Profile = (props) => {
               className={classNames.profilePic}
               alt={fullName}
             />
+            {isMember ? (
+              <div className={classNames.memberStatus}>
+                <img
+                  alt="info icon"
+                  src="icons/info.png"
+                  width={20}
+                  height={20}
+                />
+                <span>User is a Member</span>
+              </div>
+            ) : (
+              <div className={classNames.memberStatus}>
+                <img
+                  alt="info icon"
+                  src="icons/info.png"
+                  width={20}
+                  height={20}
+                />
+                <span>User is not a Member</span>
+              </div>
+            )}
```

Comment:
we should only show this to only superuser

Reply:
Added Super user condition there